starter-code/css/main.css

```diff
@@ -6,3 +6,170 @@ Green: #00B172
 White: #FFF
 
 */
+html,
+body {
+    font-family: Arial, Helvetica, sans-serif;
+    margin:0px;
+    padding: 0px;
+    width: 100%;
+    height: 100%;
+}
+
+* {
+    box-sizing: border-box;
+}
+
+.navbar{
+    width:100%;
+    padding: 20px 30px;
+    position: fixed;
+    display: flex;
+    justify-content: space-between;
+    background-color: white;
+    z-index: 1;
+}
+
+.spotify-logo{
+    height: 45px;
+}
+
+@keyframes logoSpotify {
```

Comment:
Ole esa animación guapa! Buena iniciativa tio 🔥 ¿se te ocurre cómo hacer para que su eje de origen no vaya ampliandolo desde arriba a la izquierda? Un transform: scale podría arreglar eso creo. (not 100% sure)